Src/BootCamp.Chapter/Account.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    class Account
+    {
+        private string _name;
+        private decimal[] _balance;
+
+        public string GetName()
+        {
+            return _name;
+        }
+        public void SetName(string name)
+        {
+            _name = name;
+        }
+        public decimal[] GetBalance()
+        {
+            return _balance;
+        }
+        public void SetBalance(decimal[] balance)
+        {
+            _balance = balance;
+        }
+
+        public Account(string nameAndBalance)
```

Comment:
Prefer to build an account from a name and balances. You can still make a utility class called AccountParser though where this logic is implemented. What I said makes more sense, because you don't want to couple Account to just that one implementation.